### DISCUSSION TITLE
LinkedIn OIDC migration - 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by STR1234** [supabase/supabase#19904] [Questions]
Hey Supabase Community,

I just migrated our SvelteKit Frontend LinkedIn Social Auth to the new PKCE flow. Now when I am testing the signup/login I get redirected properly but receive a 500 Internal Server Error on the LinkedIn redirect and land on **https://api.linkedin.com/oauth/v2/authorization?_l=en_US**

Does anyone else experience this or might be able to help? 

I checked the code and followed the migration closely - also checked the configuration. Everything seems to be in order on our side and as I do only get the response that internally something went wrong on LinkedIn side I do not have an idea, where to start.
Already wrote the LinkedIn support but do not know, if anybody is going to look into this.

Hopefully anybody has an idea :)

Have great holidays 🙌

**Comment by J0** [comment]
Hey @STR1234,

Sorry to hear about the issues - could we trouble you to [open a ticket](https://supabase.com/dashboard/support/new) if you are still facing the issue?

Let us know

**Comment by J0** [comment]
Closing, but lmk if the issue persists
